docs/user/installation.rst

```diff
@@ -50,7 +50,7 @@ To get ``shaarli-client`` sources and install it in a new `virtualenv`_:
   $ source ~/.virtualenvs/shaarli/bin/activate
 
   # build and install shaarli-client
-  (shaarli) $ python setup.py install
+  (shaarli) $ make install
```

Comment:
see previous comment regarding Python idioms; we should stick with using `setup.py`